crates/aptos-crypto/src/bls12381.rs

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module provides APIs for Boneh-Lynn-Shacham (BLS) multi-signatures on top of Barreto-Lynn-Scott
```

Comment:
this is great documentation! idiomatic structure is to have it in bls12381/mod.rs

Reply:
it may be worth calling out we use the min_pk version that minimize the public key size